src/screens/AdminPortal/ManageTag/editModal/EditUserTagModal.spec.tsx

```
@@ -1,14 +1,12 @@
 import React from 'react';
-import { render, screen, waitFor, cleanup } from '@testing-library/react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
```

ESLint

[error] 'fireEvent' import from '@testing-library/react' is restricted. Tests in this file use fireEvent for user interactions; use userEvent from `@testing-library/user-event` instead.

(no-restricted-imports)